### PR TITLE
Jupyter-lab: Upgrade auth proxy image

### PR DESCRIPTION
**Pull request by r4vi**
upgrade to auth proxy that supports _tunneling_. Add environment variable to the
deployment to enable the feature.

### CODE REVIEW

**Comment by r4vi** [comment]
change to platform guidance incoming to explain how to use this (just going to record some animated gifs)

File: charts/jupyter-lab/templates/deployment.yaml [hunk 1]
```diff
@@ -66,6 +66,10 @@ spec:
               value: {{ .Values.Username }}
             - name: COOKIE_MAXAGE
               value: {{ .Values.authProxy.cookie_maxage | quote }}
+            - name: TUNNEL_ENABLED
+              value: {{ .Values.authProxy.tunnel.enabled | quote }}
+            - name: TUNNEL_PORT_RANGE
+              value: {{ .Values.authProxy.tunnel.port_range | quote }}
```
Comment (xoen) [inline charts/jupyter-lab/templates/deployment.yaml]: Sorry to be late to the party. Helm convention is to use camelCase for Values, e.g. `.Values.authProxy.tunnel.portRange`. Don't know if you're still in time to change.

Comment (r4vi) [reply charts/jupyter-lab/templates/deployment.yaml]: yeah but I'm keeping consistent with the other values which are bad_cased

Comment (xoen) [reply charts/jupyter-lab/templates/deployment.yaml]: 😆

**Pull request merged**